lib/typings/cv.d.ts

```diff
@@ -19,6 +19,14 @@ export interface HistAxes {
   ranges: number[];
 }
 
+export function accumulate(dst: Mat, src: Mat, mask?: Mat): Mat;
+export function accumulateAsync(dst: Mat, src: Mat, mask?: Mat): Promise<Mat>;
+export function accumulateProduct(dst: Mat, src1: Mat, src2: Mat, mask?: Mat): Mat;
+export function accumulateProductAsync(dst: Mat, src1: Mat, src2:Mat, mask?: Mat): Promise<Mat>;
+export function accumulateSquare(dst: Mat, src: Mat, mask?: Mat): Mat;
+export function accumulateSquareAsync(dst: Mat, src: Mat, mask?: Mat): Promise<Mat>;
+export function accumulateWeighted(dst: Mat, src: Mat, alpha: number, mask?: Mat): Mat;
+export function accumulateWeightedAsync(dst: Mat, src: Mat, alpha: number, mask?: Mat): Promise<Mat>;
```

Comment:
```
export function accumulate(src: Mat, dst: Mat, mask?: Mat): Mat;
export function accumulateAsync(src: Mat, dst: Mat, mask?: Mat): Promise<Mat>;
export function accumulateProduct(src1: Mat, src2: Mat, dst: Mat, mask?: Mat): Mat;
export function accumulateProductAsync(src1: Mat, src2:Mat, dst: Mat, mask?: Mat): Promise<Mat>;
export function accumulateSquare(src: Mat, dst: Mat, mask?: Mat): Mat;
export function accumulateSquareAsync(src: Mat, dst: Mat, mask?: Mat): Promise<Mat>;
export function accumulateWeighted(src: Mat, dst: Mat, alpha: number, mask?: Mat): Mat;
export function accumulateWeightedAsync(src: Mat, dst: Mat, alpha: number, mask?: Mat): Promise<Mat>;
```

Reply:
Method chaining is possible by changing the positions of dst and src and returning dst.
I wanted to keep item 3 of the API Design guide.

method chaining example >
src1 = new cv.Mat([[1]], cv.CV_8UC1)
src2 = new cv.Mat([[2]], cv.CV_8UC1)
src3 = new cv.Mat([[3]], cv.CV_8UC1)
dst = new cv.Mat([[0]], cv.CV_32FC1)
dst.accumulate(src1).accumulate(src2).accumulate(src3)

thanks to review and I'll wait for your feedback.